webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,46 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+
+export function notify(message, type, sticky) {
```

Comment:
@waldenraines @gailsteiger I would expect it to support the same interface we use for the notification drawer... ? ```(actions: {links: [{href:, title:...?}] } ```